generator/generator.go

```diff
@@ -30,12 +31,31 @@ import (
 	"github.com/openconfig/ygot/ygen"
 )
 
+const (
+	// enumMapFn is the filename to be used for the enum map when Go code is output to a directory.
+	enumMapFn = "enum_map.go"
+	// enumFn is the filename to be used for the enum code when Go code is output to a directory.
+	enumFn = "enum.go"
+	// schemaFn is the filename to be used for the schema code when outputting to a directory.
+	schemaFn = "schema.go"
+	// interfaceFn is the filename to be used for interface code when outputting to a directory.
+	interfaceFn = "union.go"
```

Comment:
Why is the file that contains interface code called union? Some terminology I am not familiar with maybe?

Reply:
When ygen generates any interface it is because it is for a YANG union - a bit like protobuf oneof. So it seemed like a reasonable split here was to say that this file was named after the YANG construct that it contained. I'm agnostic on this.